client/src/produkter/tabs/AboutTab.tsx

```diff
@@ -1,199 +1,293 @@
-                >
-                  Lagre
-                </Button>
-              </>
-            )}
-          </VStack>
-        </VStack>
-      </form>
-    </Tabs.Panel>
-  );
+const AboutTab = ({series, onSubmit, isoCategory, isEditable, showInputError}: Props) => {
+    const formMethods = useFormContext<EditSeriesInfo>();
+    const [showEditDescriptionMode, setShowEditDescriptionMode] = useState(false);
+    const [showEditKeywordsMode, setShowEditKeywordsMode] = useState(false);
+    const [showEditUrlMode, setShowEditUrlMode] = useState(false);
+    const formRef = useRef<HTMLFormElement>(null);
+    const [urlFormatError, setUrlFormatError] = useState<string | undefined>(undefined);
+    const [keywordFormatError, setKeywordFormatError] = useState<string | undefined>(undefined);
+    const [inputValue, setInputValue] = useState("");
+    const [selectedOptions, setSelectedOptions] = useState<string[]>(series.seriesData.attributes.keywords ? series.seriesData.attributes.keywords : []);
+
+    const validKeywordLetters = new RegExp(/^[A-Za-zÀ-ÖØ-öø-ÿ0-9_\s]*$/);
+
+    const getDescription = () => (
+        <>
+            Beskrivelsen vises på produktsiden og bør:
+            <ul>
+                <li>Være mellom 100 og 750 tegn</li>
+                <li>Ikke inneholde tekniske data, for eksempel. “Totalvekt i str. 42x40”</li>
+                <li>Ha en nøytral språkstil uten markedsføringsuttrykk</li>
+            </ul>
+        </>
+    );
+
+    const handleSaveDescription = () => {
+        setShowEditDescriptionMode(false);
+        formRef.current?.requestSubmit();
+    };
+
+    const handleSaveUrl = () => {
+        setUrlFormatError(undefined);
+        setShowEditUrlMode(false);
+        formRef.current?.requestSubmit();
+    };
+
+    const handleSaveKeywords = () => {
+        setKeywordFormatError(undefined);
+        setShowEditKeywordsMode(false);
+        if (selectedOptions.length <= 3) {
+            formMethods.setValue("keywords", selectedOptions)
+            formRef.current?.requestSubmit();
+        } else setKeywordFormatError("Du kan maksimalt velge 3 nøkkelord");
+    };
+
+    return (
+        <Tabs.Panel value="about" className="tab-panel">
+            <form method="POST" onSubmit={formMethods.handleSubmit(onSubmit)} ref={formRef}>
+                <VStack gap="14">
+                    <VStack gap="2">
+                        <Heading level="2" size="small">
+                            Iso-kategori (kode)
+                        </Heading>
+
+                        <div>
+                            {isoCategory?.isoTitle} ({isoCategory?.isoCode})
+                        </div>
+                    </VStack>
+
+                    <VStack gap="2">
+                        <Heading level="2" size="small">
+                            {labelRequired("Produktbeskrivelse")}
+                        </Heading>
+
+                        {!showEditDescriptionMode && (
```

Comment:
Tenker at vi kan skille ut dei tre bolkane her, description, url og keywords i egne komponentar  for å gjere koden meire lesbar på sikt. Behøver ikkje gjere det nå, men du kan gjerne det om du har tid til overs.